canonical-websites/www.ubuntu.com
# Add MAAS user survey data privacy page

## Done

**Asking legal if this shouldn't be made more generic**

- Add MAAS user survey data privacy page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/data-privacy/snap-store](http://0.0.0.0:8001/legal/data-privacy/snap-store)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1XKWK3VvJe0Z_soJqO3F0BrrXwlP9ZObYYKxA1hym7QU/edit)


## Issue / Card

Fixes #4666
